app/src/test/report/actions/UserActionTests.ts

```diff
@@ -0,0 +1,36 @@
+import {expect} from "chai";
+
+import {Sandbox} from "../../Sandbox";
+import {createMockStore} from "../../mocks/mockStore";
+import {UserTypeKeys} from "../../../main/report/actionTypes/UsersActionTypes";
+import {userActions} from "../../../main/report/actions/userActions";
+import {UserService} from "../../../main/report/services/UserService";
+import {mockUser} from "../../mocks/mockModels";
+
+describe("User actions tests", () => {
+    const sandbox = new Sandbox();
+    let store: any = null;
+
+    beforeEach(() => {
+        store = createMockStore();
+    });
+
+    afterEach(() => {
+        sandbox.restore();
+    });
+
+    it("dispatches report readers fetched action if get report readers action is dispatched", (done) => {
+        const fakeUser = mockUser();
+        sandbox.setStubFunc(UserService.prototype, "getReportReaders", () => {
+            return Promise.resolve([fakeUser]);
+        });
+        store.dispatch(userActions.getReportReaders("test"));
+        setTimeout(() => {
```

Comment:
You should use `checkAsync` here - it handles calling `done` for you, even in the case where an exception is thrown, where this code will just hang.